waspc/src/Wasp/AppSpec/Entity.hs

```diff
@@ -30,6 +31,9 @@ data Entity = Entity
 
 instance IsDecl Entity
 
+instance FromJSON Entity where
```

Comment:
We don't allow defining entities via Wasp file any more, but we left some of the old machinery in for now since it was easier then purging it, so I just put this `error` here, I think this should be quite fine for now.

Reply:
I think this is fine for now.